server/services/flightRecord.js

```diff
@@ -11,7 +12,34 @@ const createFlightRecord = async (flightData) => {
       airplaneId: airplane._id,
       flightDay,
       flightId: flightData.flightId,
+      businessTickets: flightData.businessTickets,
+      standartTickets: flightData.standartTickets,
+      economTickets: flightData.economTickets,
     });
+    for (let i = 0; i < flightData.businessTickets; i++) {
+      const newTicket = {
+        price: flightData.businessPrice,
+        flightRecordId: newFlight._id,
+        ticketClass: "business",
+      };
+      await Ticket.create(newTicket);
+    }
+    for (let i = 0; i < flightData.standartTickets; i++) {
+      const newTicket = {
+        price: flightData.standartPrice,
+        flightRecordId: newFlight._id,
+        ticketClass: "standart",
+      };
+      await Ticket.create(newTicket);
+    }
+    for (let i = 0; i < flightData.economTickets; i++) {
```

Comment:
could you move this 3 loops in one function/method?

Reply:
yes sure